CoreDistributedCache/NHibernate.Caches.CoreDistributedCache.Memcached/NHibernate.Caches.CoreDistributedCache.Memcached.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../NHibernate.Caches.props" />
+  <PropertyGroup>
+    <Product>NHibernate.Caches.CoreDistributedCache.Memcached</Product>
+    <Title>NHibernate.Caches.CoreDistributedCache.Memcached</Title>
+    <Description>Memcached cache provider for NHibernate using .Net Core IDistributedCache (EnyimMemcachedCore).</Description>
+    <!-- Targeting net461 explicitly in order to avoid https://github.com/dotnet/standard/issues/506 for net461 consumers-->
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);3001;3002</NoWarn>
+    <SignAssembly>False</SignAssembly>
```

Comment:
EnyimMemcachedCore does not have strong name. This creates no issues for netcoreapp2.0 tests and compilation whatever the target, but if we sign NHibernate.Caches.CoreDistributedCache.Memcached, then net461 tests fail loading the dependency, complaining `System.IO.FileLoadException : Could not load file or assembly 'EnyimMemcachedCore, Version=2.1.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required.`

It seems [it is recommended to no more strongly name assemblies in most cases](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md), so not signing this package should be acceptable.